### PR TITLE
update pulp-repo-versions.yml

### PR DESCRIPTION
**Pull request by scrungus**
update pulp repo versions due to missing snapshot for rocky 9.2

### CODE REVIEW

File: etc/kayobe/pulp-repo-versions.yml [hunk 1]
```diff
@@ -44,12 +44,13 @@ stackhpc_pulp_repo_rocky_9_1_baseos_version: 20230228T044432
...
+stackhpc_pulp_repo_rocky_9_2_appstream_version: 20230627T185020
+stackhpc_pulp_repo_rocky_9_2_baseos_version: 20230627T185020
+stackhpc_pulp_repo_rocky_9_2_crb_version: 20230627T185020
+stackhpc_pulp_repo_rocky_9_2_extras_version: 20230612T072506
+stackhpc_pulp_repo_rocky_9_2_highavailability_version: 20230628T031859
 stackhpc_pulp_repo_treasuredata_4_version: 20221105T035018
 stackhpc_pulp_repo_ubuntu_cloud_archive_version: 20230607T200208
 stackhpc_pulp_repo_ubuntu_focal_security_version: 20230607T200208
 stackhpc_pulp_repo_ubuntu_focal_version: 20230607T200208
+
```
Comment (jovial) [inline etc/kayobe/pulp-repo-versions.yml]: White-space change, but end of file whitespace is more correct: https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline

**Review by jovial — DISMISSED** [review]

File: etc/kayobe/pulp-repo-versions.yml (hunk 1 above)
Comment (jovial) [reply etc/kayobe/pulp-repo-versions.yml]: OK, tox doesn't like it. I guess we should remove.

**Comment by mnasiadka** [comment]
Shouldn't we have a reno for that?

File: etc/kayobe/pulp-repo-versions.yml (hunk 1 above)
Comment (jovial) [reply etc/kayobe/pulp-repo-versions.yml]:
```suggestion
```

**Review by jovial — APPROVED** [review]
Looks good. Please run: https://github.com/stackhpc/stackhpc-release-train/actions/workflows/package-promote.yml and link to the job before merging.

**Comment by Alex-Welsh** [comment]
CI is looking particularly grumpy today. If we want to test this change, I do have a fix for Rocky 9 [here](https://github.com/stackhpc/stackhpc-kayobe-config/actions/runs/5399655931)

**Comment by scrungus** [comment]
https://github.com/stackhpc/stackhpc-release-train/actions/runs/5401285324

**Comment by markgoddard** [comment]
> Looks good. Please run: https://github.com/stackhpc/stackhpc-release-train/actions/workflows/package-promote.yml and link to the job before merging.

This happens automatically for packages, it only needs to be done for containers.

**Comment by Alex-Welsh** [comment]
Tests should work now.

**Pull request merged**